osu.Game/Online/Multiplayer/MultiplayerRoomUser.cs

```diff
@@ -27,6 +31,13 @@ public class MultiplayerRoomUser : IEquatable<MultiplayerRoomUser>
         [Key(2)]
         public BeatmapAvailability BeatmapAvailability { get; set; } = BeatmapAvailability.LocallyAvailable();
 
+        /// <summary>
+        /// Any mods applicable only to the local user.
+        /// </summary>
+        [Key(3)]
+        [NotNull]
+        public IEnumerable<APIMod> UserMods { get; set; } = Enumerable.Empty<APIMod>();
```

Comment:
Call this `OptionalMods` or just `Mods`? Don't think the user prefix is required when this whole class is applicable to a user.

Reply:
Went with `Mods`.